srcpkgs/apr/template

```diff
@@ -29,14 +29,21 @@ if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 	LDFLAGS="-latomic"
 fi
 
+do_check() {
+	cp "${FILESDIR}/etc-services" "${XBPS_MASTERDIR}/etc/services"
+	make test
+}
```

Comment:
Would installing the default /etc/services file by adding iana-etc to checkdepends work?

Reply:
Aha, yes it does. Thank you. Fixed.